src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Utilities;
+using Python.Runtime;
+
+namespace DSCPython
+{
+    [SupressImportIntoVM]
+    public enum EvaluationState { Begin, Success, Failed }
```

Comment:
if this needs to be public can we add summary tags?
If not can we make it internal?